examples/fluid/dam_break_2d.jl

```diff
@@ -71,6 +71,21 @@ ode = semidiscretize(semi, tspan)
 info_callback = InfoCallback(interval=100)
 saving_callback = SolutionSavingCallback(dt=0.02, prefix="")
 
+# Example on how interpolated planes can be saved using the postprocessing callback
+# function save_interpolated_plane(v, u, t, system)
+#     interpolation_start = [0.0, 0.0]
+#     interpolation_end = [tank_size[1], 3.0]
+#     resolution = 0.005
+#     file_id = Int(ceil(t*10000))
+
+#     interpolate_plane_2d_vtk(interpolation_start, interpolation_end, resolution,
+#                              semi, system, v, u, filename="plane_$file_id.vti")
```

Comment:
This is not correct! These have to be the unwrapped `v_ode` and `u_ode`. I realized that I forgot to send my comment in #424 that we don't just need a version with `v` and `u`, but also `v_ode`, `u_ode` and `semi`.